tests/fips/mozilla_nss/firefox_nss.pm

```diff
@@ -176,7 +176,7 @@ sub run {
         # Retype password again once the password missed input
         # The problem frequently happaned in aarch64
         if (check_screen("firefox-password-typefield-miss")) {
-            record_soft_failure "Firefox password is missing to input, see poo#77143";
+            record_info("aarch64 type_missing", "Firefox password is missing to input, please refer to bsc#1179749 & poo#105343");
```

Comment:
remove poo#77143 is ok
but added bsc#1179749 & poo#105343 is useless too, as the bug is fixed and the poo will be resolved too.

Reply:
yes, because it is just a reference if the issue happened again randomly next time. It will have a record to check which bug or poo were reported before. is that ok?

Reply:
Sure, it is ok.